### PR TITLE
    add jquery ui autocomplete to object ip tab ( Allocations ) OS Interface

### PR DESCRIPTION
**Pull request by github138**
lists object port names as OS Interface

    * renderIPForObject (): add autocomplete
    * getAutocompleteListAJAX (): add realms "bond_name", use label/value results
    * popup (): use label/value result

### CODE REVIEW

File: wwwroot/inc/interface.php [hunk 1]
```diff
@@ -1882,6 +1882,26 @@ function printNewItemTR ($default_type)
 		echo "</td></tr></form>";
 	}
 	global $aat;
+
+	includeJQueryUI (TRUE);
```
Comment (andriyanov) [inline wwwroot/inc/interface.php]: could you please move this code into printNewItemTR ?

File: wwwroot/inc/interface.php [hunk 2]
```diff
@@ -3396,6 +3396,31 @@ function printNewItemTR ()
 	}
 	global $aat;
 
+	includeJQueryUI (TRUE);
```
Comment (andriyanov) [inline wwwroot/inc/interface.php]: could you please move this code into printNewItemTR ?

**Comment by github138** [comment]
moved code and also fixed autocomplete for existing allocations.

**Comment by andriyanov** [comment]
Merged, thank you. Could you provide the changelog line so I can add an announcement for users into Changelog?

**Comment by github138** [comment]
update: add autocomplete to OS interface fields

That ok?

**Pull request merged**